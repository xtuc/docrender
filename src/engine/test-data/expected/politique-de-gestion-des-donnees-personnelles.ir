(
    content: [
        Clear,
        Title((
            level: 3,
            text: Text("Politique de gestion des données personnelles"),
        ), []),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("[Nom de l\'entreprise/organisation]", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], false),
            InlineText([
                Text(" s\'engage à protéger la vie privée de ses utilisateurs et à traiter leurs données personnelles dans le respect des lois et réglementations en vigueur, notamment le Règlement Général sur la Protection des Données (RGPD)."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("1. Données collectées", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Nous collectons les données suivantes :"),
            ], true),
        ]),
        Clear,
        Clear,
        List([
            InlineText([
                TextWithOptions("Données d\'identification :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" nom, prénom, adresse email, numéro de téléphone, etc."),
            ], true),
            InlineText([
                TextWithOptions("Données de connexion :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" adresse IP, type de navigateur, système d\'exploitation, etc."),
            ], true),
            InlineText([
                TextWithOptions("Données de navigation :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" pages visitées, liens cliqués, durée de la visite, etc."),
            ], true),
            InlineText([
                TextWithOptions("Données relatives aux transactions :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" historique des achats, informations de paiement, etc. (si applicable)"),
            ], true),
            InlineText([
                TextWithOptions("Toute autre donnée que vous nous fournissez volontairement.", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("2. Finalités du traitement", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Vos données sont collectées et traitées pour les finalités suivantes :"),
            ], true),
        ]),
        Clear,
        Clear,
        List([
            InlineText([
                TextWithOptions("Fournir et améliorer nos services :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" répondre à vos demandes, personnaliser votre expérience, etc."),
            ], true),
            InlineText([
                TextWithOptions("Gérer votre compte utilisateur :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" créer et gérer votre compte, traiter vos commandes, etc. (si applicable)"),
            ], true),
            InlineText([
                TextWithOptions("Vous contacter :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" répondre à vos questions, vous envoyer des informations importantes, etc."),
            ], true),
            InlineText([
                TextWithOptions("Analyser l\'utilisation de nos services :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" améliorer nos services, développer de nouvelles fonctionnalités, etc."),
            ], true),
            InlineText([
                TextWithOptions("Respecter nos obligations légales et réglementaires.", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("3. Base légale du traitement", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Le traitement de vos données repose sur les bases légales suivantes :"),
            ], true),
        ]),
        Clear,
        Clear,
        List([
            InlineText([
                TextWithOptions("Votre consentement :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" pour certaines opérations de traitement, nous vous demanderons votre consentement explicite."),
            ], true),
            InlineText([
                TextWithOptions("L\'exécution d\'un contrat :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" pour traiter vos commandes et gérer votre compte utilisateur (si applicable)."),
            ], true),
            InlineText([
                TextWithOptions("Le respect d\'une obligation légale :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" pour répondre à nos obligations légales et réglementaires."),
            ], true),
            InlineText([
                TextWithOptions("Notre intérêt légitime :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" pour améliorer nos services et vous proposer des offres pertinentes."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("4. Destinataires des données", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Vos données peuvent être transmises aux destinataires suivants :"),
            ], true),
        ]),
        Clear,
        Clear,
        List([
            InlineText([
                TextWithOptions("Nos employés et sous-traitants :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" uniquement dans la mesure nécessaire à l\'accomplissement de leurs tâches."),
            ], true),
            InlineText([
                TextWithOptions("Nos partenaires commerciaux :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" avec votre consentement préalable."),
            ], true),
            InlineText([
                TextWithOptions("Les autorités compétentes :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" en cas d\'obligation légale."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("5. Durée de conservation", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Vos données sont conservées pendant la durée nécessaire à la réalisation des finalités pour lesquelles elles ont été collectées, et en tout état de cause dans le respect des délais légaux de conservation.  "),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("6. Sécurité des données", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Nous mettons en œuvre des mesures techniques et organisationnelles appropriées pour garantir la sécurité de vos données et prévenir tout accès non autorisé, toute modification, divulgation ou destruction."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("7. Vos droits", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Vous disposez des droits suivants concernant vos données personnelles :"),
            ], true),
        ]),
        Clear,
        Clear,
        List([
            InlineText([
                TextWithOptions("Droit d\'accès :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" vous pouvez accéder aux données que nous détenons sur vous."),
            ], true),
            InlineText([
                TextWithOptions("Droit de rectification :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" vous pouvez demander la correction des données inexactes ou incomplètes."),
            ], true),
            InlineText([
                TextWithOptions("Droit à l\'effacement :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" vous pouvez demander la suppression de vos données dans certaines circonstances."),
            ], true),
            InlineText([
                TextWithOptions("Droit à la limitation du traitement :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" vous pouvez demander la limitation du traitement de vos données dans certaines circonstances."),
            ], true),
            InlineText([
                TextWithOptions("Droit à la portabilité des données :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" vous pouvez recevoir vos données dans un format portable."),
            ], true),
            InlineText([
                TextWithOptions("Droit d\'opposition :", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
                Text(" vous pouvez vous opposer au traitement de vos données dans certaines circonstances."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Pour exercer vos droits, vous pouvez nous contacter à l\'adresse email suivante : "),
            ], false),
            InlineText([
                TextWithOptions("[adresse email]", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], false),
            InlineText([
                Text("."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("8. Cookies", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Nous utilisons des cookies pour améliorer votre expérience sur notre site web. Vous pouvez configurer votre navigateur pour refuser les cookies, mais cela pourrait affecter certaines fonctionnalités du site.  "),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("9. Modifications de la politique", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Nous nous réservons le droit de modifier cette politique de gestion des données personnelles à tout moment. Les modifications seront publiées sur notre site web."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("10. Contact", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                Text("Pour toute question relative à cette politique, vous pouvez nous contacter à l\'adresse email suivante : "),
            ], false),
            InlineText([
                TextWithOptions("[adresse email]", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], false),
            InlineText([
                Text("."),
            ], true),
        ]),
        Clear,
        Paragraph([
            InlineText([
                TextWithOptions("[Date de la dernière mise à jour]", (
                    styles: [
                        ("font-weight", "bold"),
                    ],
                )),
            ], false),
        ]),
        Paragraph([]),
        Paragraph([]),
    ],
    styles: {
        "pbNORMAL_TEXT": [
            ("line-height", "1.7249999"),
        ],
    },
)