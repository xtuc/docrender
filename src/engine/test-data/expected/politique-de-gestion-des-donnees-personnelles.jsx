import { createElement, Fragment } from "https://unpkg.com/es-react";

function Heading({ style, id, level, children }) {
  const tag = "h" + level;
  return createElement(tag, { style, id }, children);
}

function Paragraph({ children }) {
  return createElement("p", {}, children);
}

function Code({ children }) {
  return createElement("code", {}, children);
}

function Image({ src }) {
  return createElement("img", { src });
}

function Link({ href }) {
  return createElement("a", { href });
}

function List({ children }) {
  return createElement("ul", {}, children);
}

function ListItem({ children }) {
  return createElement("li", {}, children);
}

export default function Doc({
  heading = Heading,
  paragraph = Paragraph,
  code = Code,
  image = Image,
  link = Link,
  list = List,
  listItem = ListItem,
} = {}) {
  return createElement(
    Fragment,
    {},
    createElement(
      heading,
      { level: 3, id: "pb-todo" },
      `Politique de gestion des données personnelles`,
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `[Nom de l'entreprise/organisation]`,
      ),
      ` s'engage à protéger la vie privée de ses utilisateurs et à traiter leurs données personnelles dans le respect des lois et réglementations en vigueur, notamment le Règlement Général sur la Protection des Données (RGPD).`,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `1. Données collectées`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Nous collectons les données suivantes :`,
      `
`,
    ),
    createElement(
      list,
      {},
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Données d'identification :`,
        ),
        `
`,
        ` nom, prénom, adresse email, numéro de téléphone, etc.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Données de connexion :`,
        ),
        `
`,
        ` adresse IP, type de navigateur, système d'exploitation, etc.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Données de navigation :`,
        ),
        `
`,
        ` pages visitées, liens cliqués, durée de la visite, etc.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Données relatives aux transactions :`,
        ),
        `
`,
        ` historique des achats, informations de paiement, etc. (si applicable)`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Toute autre donnée que vous nous fournissez volontairement.`,
        ),
        `
`,
      ),
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `2. Finalités du traitement`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Vos données sont collectées et traitées pour les finalités suivantes :`,
      `
`,
    ),
    createElement(
      list,
      {},
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Fournir et améliorer nos services :`,
        ),
        `
`,
        ` répondre à vos demandes, personnaliser votre expérience, etc.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Gérer votre compte utilisateur :`,
        ),
        `
`,
        ` créer et gérer votre compte, traiter vos commandes, etc. (si applicable)`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Vous contacter :`,
        ),
        `
`,
        ` répondre à vos questions, vous envoyer des informations importantes, etc.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Analyser l'utilisation de nos services :`,
        ),
        `
`,
        ` améliorer nos services, développer de nouvelles fonctionnalités, etc.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Respecter nos obligations légales et réglementaires.`,
        ),
        `
`,
      ),
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `3. Base légale du traitement`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Le traitement de vos données repose sur les bases légales suivantes :`,
      `
`,
    ),
    createElement(
      list,
      {},
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Votre consentement :`,
        ),
        `
`,
        ` pour certaines opérations de traitement, nous vous demanderons votre consentement explicite.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `L'exécution d'un contrat :`,
        ),
        `
`,
        ` pour traiter vos commandes et gérer votre compte utilisateur (si applicable).`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Le respect d'une obligation légale :`,
        ),
        `
`,
        ` pour répondre à nos obligations légales et réglementaires.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Notre intérêt légitime :`,
        ),
        `
`,
        ` pour améliorer nos services et vous proposer des offres pertinentes.`,
        `
`,
      ),
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `4. Destinataires des données`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Vos données peuvent être transmises aux destinataires suivants :`,
      `
`,
    ),
    createElement(
      list,
      {},
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Nos employés et sous-traitants :`,
        ),
        `
`,
        ` uniquement dans la mesure nécessaire à l'accomplissement de leurs tâches.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Nos partenaires commerciaux :`,
        ),
        `
`,
        ` avec votre consentement préalable.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Les autorités compétentes :`,
        ),
        `
`,
        ` en cas d'obligation légale.`,
        `
`,
      ),
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `5. Durée de conservation`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Vos données sont conservées pendant la durée nécessaire à la réalisation des finalités pour lesquelles elles ont été collectées, et en tout état de cause dans le respect des délais légaux de conservation.  `,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `6. Sécurité des données`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Nous mettons en œuvre des mesures techniques et organisationnelles appropriées pour garantir la sécurité de vos données et prévenir tout accès non autorisé, toute modification, divulgation ou destruction.`,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement("span", { style: { fontWeight: "bold" } }, `7. Vos droits`),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Vous disposez des droits suivants concernant vos données personnelles :`,
      `
`,
    ),
    createElement(
      list,
      {},
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Droit d'accès :`,
        ),
        `
`,
        ` vous pouvez accéder aux données que nous détenons sur vous.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Droit de rectification :`,
        ),
        `
`,
        ` vous pouvez demander la correction des données inexactes ou incomplètes.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Droit à l'effacement :`,
        ),
        `
`,
        ` vous pouvez demander la suppression de vos données dans certaines circonstances.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Droit à la limitation du traitement :`,
        ),
        `
`,
        ` vous pouvez demander la limitation du traitement de vos données dans certaines circonstances.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Droit à la portabilité des données :`,
        ),
        `
`,
        ` vous pouvez recevoir vos données dans un format portable.`,
        `
`,
      ),
      createElement(
        listItem,
        {},
        createElement(
          "span",
          { style: { fontWeight: "bold" } },
          `Droit d'opposition :`,
        ),
        `
`,
        ` vous pouvez vous opposer au traitement de vos données dans certaines circonstances.`,
        `
`,
      ),
    ),
    createElement(
      paragraph,
      {},
      `Pour exercer vos droits, vous pouvez nous contacter à l'adresse email suivante : `,
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `[adresse email]`,
      ),
      `.`,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement("span", { style: { fontWeight: "bold" } }, `8. Cookies`),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Nous utilisons des cookies pour améliorer votre expérience sur notre site web. Vous pouvez configurer votre navigateur pour refuser les cookies, mais cela pourrait affecter certaines fonctionnalités du site.  `,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `9. Modifications de la politique`,
      ),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Nous nous réservons le droit de modifier cette politique de gestion des données personnelles à tout moment. Les modifications seront publiées sur notre site web.`,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement("span", { style: { fontWeight: "bold" } }, `10. Contact`),
      `
`,
    ),
    createElement(
      paragraph,
      {},
      `Pour toute question relative à cette politique, vous pouvez nous contacter à l'adresse email suivante : `,
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `[adresse email]`,
      ),
      `.`,
      `
`,
    ),
    createElement(
      paragraph,
      {},
      createElement(
        "span",
        { style: { fontWeight: "bold" } },
        `[Date de la dernière mise à jour]`,
      ),
    ),
    createElement(paragraph, {}),
    createElement(paragraph, {}),
  );
}
