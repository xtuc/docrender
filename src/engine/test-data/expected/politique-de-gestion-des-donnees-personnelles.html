<style>
  .pbNORMAL_TEXT {
    line-height: 1.7249999;
  }
</style>
<h3 id="pb-Politique%20de%20gestion%20des%20donn%C3%A9es%20personnelles">
  Politique de gestion des données personnelles
</h3>
<p>
  <span style="font-weight: bold">[Nom de l'entreprise/organisation]</span>
  s'engage à protéger la vie privée de ses utilisateurs et à traiter leurs
  données personnelles dans le respect des lois et réglementations en vigueur,
  notamment le Règlement Général sur la Protection des Données (RGPD).
</p>
<p><span style="font-weight: bold">1. Données collectées</span></p>
<p>Nous collectons les données suivantes :</p>
<ul>
  <li>
    <span style="font-weight: bold">Données d'identification :</span>
    nom, prénom, adresse email, numéro de téléphone, etc.
  </li>
  <li>
    <span style="font-weight: bold">Données de connexion :</span>
    adresse IP, type de navigateur, système d'exploitation, etc.
  </li>
  <li>
    <span style="font-weight: bold">Données de navigation :</span>
    pages visitées, liens cliqués, durée de la visite, etc.
  </li>
  <li>
    <span style="font-weight: bold">Données relatives aux transactions :</span>
    historique des achats, informations de paiement, etc. (si applicable)
  </li>
  <li>
    <span style="font-weight: bold"
      >Toute autre donnée que vous nous fournissez volontairement.</span
    >
  </li>
</ul>
<p><span style="font-weight: bold">2. Finalités du traitement</span></p>
<p>Vos données sont collectées et traitées pour les finalités suivantes :</p>
<ul>
  <li>
    <span style="font-weight: bold">Fournir et améliorer nos services :</span>
    répondre à vos demandes, personnaliser votre expérience, etc.
  </li>
  <li>
    <span style="font-weight: bold">Gérer votre compte utilisateur :</span>
    créer et gérer votre compte, traiter vos commandes, etc. (si applicable)
  </li>
  <li>
    <span style="font-weight: bold">Vous contacter :</span>
    répondre à vos questions, vous envoyer des informations importantes, etc.
  </li>
  <li>
    <span style="font-weight: bold"
      >Analyser l'utilisation de nos services :</span
    >
    améliorer nos services, développer de nouvelles fonctionnalités, etc.
  </li>
  <li>
    <span style="font-weight: bold"
      >Respecter nos obligations légales et réglementaires.</span
    >
  </li>
</ul>
<p><span style="font-weight: bold">3. Base légale du traitement</span></p>
<p>Le traitement de vos données repose sur les bases légales suivantes :</p>
<ul>
  <li>
    <span style="font-weight: bold">Votre consentement :</span>
    pour certaines opérations de traitement, nous vous demanderons votre
    consentement explicite.
  </li>
  <li>
    <span style="font-weight: bold">L'exécution d'un contrat :</span>
    pour traiter vos commandes et gérer votre compte utilisateur (si
    applicable).
  </li>
  <li>
    <span style="font-weight: bold">Le respect d'une obligation légale :</span>
    pour répondre à nos obligations légales et réglementaires.
  </li>
  <li>
    <span style="font-weight: bold">Notre intérêt légitime :</span>
    pour améliorer nos services et vous proposer des offres pertinentes.
  </li>
</ul>
<p><span style="font-weight: bold">4. Destinataires des données</span></p>
<p>Vos données peuvent être transmises aux destinataires suivants :</p>
<ul>
  <li>
    <span style="font-weight: bold">Nos employés et sous-traitants :</span>
    uniquement dans la mesure nécessaire à l'accomplissement de leurs tâches.
  </li>
  <li>
    <span style="font-weight: bold">Nos partenaires commerciaux :</span>
    avec votre consentement préalable.
  </li>
  <li>
    <span style="font-weight: bold">Les autorités compétentes :</span>
    en cas d'obligation légale.
  </li>
</ul>
<p><span style="font-weight: bold">5. Durée de conservation</span></p>
<p>
  Vos données sont conservées pendant la durée nécessaire à la réalisation des
  finalités pour lesquelles elles ont été collectées, et en tout état de cause
  dans le respect des délais légaux de conservation.
</p>
<p><span style="font-weight: bold">6. Sécurité des données</span></p>
<p>
  Nous mettons en œuvre des mesures techniques et organisationnelles appropriées
  pour garantir la sécurité de vos données et prévenir tout accès non autorisé,
  toute modification, divulgation ou destruction.
</p>
<p><span style="font-weight: bold">7. Vos droits</span></p>
<p>Vous disposez des droits suivants concernant vos données personnelles :</p>
<ul>
  <li>
    <span style="font-weight: bold">Droit d'accès :</span>
    vous pouvez accéder aux données que nous détenons sur vous.
  </li>
  <li>
    <span style="font-weight: bold">Droit de rectification :</span>
    vous pouvez demander la correction des données inexactes ou incomplètes.
  </li>
  <li>
    <span style="font-weight: bold">Droit à l'effacement :</span>
    vous pouvez demander la suppression de vos données dans certaines
    circonstances.
  </li>
  <li>
    <span style="font-weight: bold">Droit à la limitation du traitement :</span>
    vous pouvez demander la limitation du traitement de vos données dans
    certaines circonstances.
  </li>
  <li>
    <span style="font-weight: bold">Droit à la portabilité des données :</span>
    vous pouvez recevoir vos données dans un format portable.
  </li>
  <li>
    <span style="font-weight: bold">Droit d'opposition :</span>
    vous pouvez vous opposer au traitement de vos données dans certaines
    circonstances.
  </li>
</ul>
<p>
  Pour exercer vos droits, vous pouvez nous contacter à l'adresse email suivante
  : <span style="font-weight: bold">[adresse email]</span>.
</p>
<p><span style="font-weight: bold">8. Cookies</span></p>
<p>
  Nous utilisons des cookies pour améliorer votre expérience sur notre site web.
  Vous pouvez configurer votre navigateur pour refuser les cookies, mais cela
  pourrait affecter certaines fonctionnalités du site.
</p>
<p><span style="font-weight: bold">9. Modifications de la politique</span></p>
<p>
  Nous nous réservons le droit de modifier cette politique de gestion des
  données personnelles à tout moment. Les modifications seront publiées sur
  notre site web.
</p>
<p><span style="font-weight: bold">10. Contact</span></p>
<p>
  Pour toute question relative à cette politique, vous pouvez nous contacter à
  l'adresse email suivante :
  <span style="font-weight: bold">[adresse email]</span>.
</p>
<p><span style="font-weight: bold">[Date de la dernière mise à jour]</span></p>
